interfaces/package.json

```diff
@@ -19,6 +19,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "concord": "~0.2.2",
```

Comment:
Okay, apparently `npx` will run the version in `node_modules` if exists, and download and install latest if not exists.

The missing piece for it to properly work is that `generate_interfaces` script usually runs before `rush install`, so it will still use the latest since no package in node_modules.

Reply:
It's a longer piece of spaghetti than that, though.  `generate_interfaces` bypasses rush there (again!) and `npm install`s.  https://github.com/binaris/shiftjs/blob/0aced8b806f09eb9044422bfe80ec3079450ebad/scripts/generate_interfaces#L4-L6
Yessss.  I am not proud of myself for this.